src/features/users/user-controller.ts

```diff
@@ -9,12 +10,18 @@ import Session from '../../database/models/session';
 import moment = require('moment');
 import configurations from '../../configurations';
 import NoAssociatedUniversityError from '../../exceptions/no-associated-university-error';
-import { UniqueConstraintError } from 'sequelize';
+import { UniqueConstraintError, Op } from 'sequelize';
```

Comment:
What is `Op`? I don't see it being used here.

Reply:
Good catch, I removed the code that used it, i'll remove that in the future